src/schema/metadata/ElectrodeManufacturer.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: ElectrodeManufacturer
+description: |
+  Can be used if all electrodes are of the same manufacturer
+  (for example, AD-TECH, DIXI).
```

Comment:
```suggestion
  (for example, `"AD-TECH, DIXI"`).
```

Reply:
Should this be `"AD-TECH, DIXI"` or `"AD-TECH", "DIXI"`?

Reply:
Same here @dorahermes can you help us what this example should be?

Reply:
It looks like they're different manufacturers rather than a single comma-connected name, so I'm going to go with the latter option.

Reply:
okay, thanks 👍